File: HackerRank/30-Days-of-Code/C++/day05-Loops.cpp
#include <iostream>

using namespace std;

// Question Link : https://www.hackerrank.com/challenges/30-loops/problem?isFullScreen=true


int main()
{
    int n;
    cin>>n;
    for(int i=1;i<=10;i++){
        cout << n << " x " << i << " = " << n*i << endl;
    }

    return 0;
}

File: HackerRank/30-Days-of-Code/Java/day05Loops.java
import java.util.Scanner;

// Question Link : https://www.hackerrank.com/challenges/30-loops/problem?isFullScreen=true

public class day05Loops {
    public static void main(String[] args) {
       Scanner scan = new Scanner(System.in);
       int n =scan.nextInt();
       for(int i=1;i<=10;i++){
        System.out.println(n + " x " + i +" = " + n*i);
       }
       
       
       scan.close();
    }
}
